ACM#9626: Update the  command params (#6097)
open-cluster-management/rhacm-docs: clusters/hosted_control_planes/scale_nodepool_hosted_bm.adoc

[#scaling-the-nodepool]
= Scaling the NodePool object for a hosted cluster

> You add nodes to your hosted cluster by scaling the `NodePool` object.

Reviewer: ```suggestion
To add nodes to your hosted cluster, scale the `NodePool` object. Complete the following steps:
```

Author: Suggestion is addressed in https://github.com/stolostron/rhacm-docs/pull/6139

clusteroperator.config.openshift.io/openshift-controller-manager               4.12z    True        False         False      40m
clusteroperator.config.openshift.io/openshift-samples                          4.12z    True        False         False      8m54s
clusteroperator.config.openshift.io/operator-lifecycle-manager                 4.12z    True        False         False      40m
clusteroperator.config.openshift.io/operator-lifecycle-manager-catalog         4.12z    True        False         False      40m
clusteroperator.config.openshift.io/operator-lifecycle-manager-packageserver   4.12z    True        False         False      40m
clusteroperator.config.openshift.io/service-ca                                 4.12z    True        False         False      11m
clusteroperator.config.openshift.io/storage                                    4.12z    True        False         False      11m
----

[#adding-nodepool-bm]
== Adding node pools

You can create node pools for a hosted cluster by specifying a name, number of replicas, and any additional information, such as an agent label selector.

> . To create a node pool, enter the following information. In this example, the node pool uses agents with the 'size=large,zone notin (az1,az2)' label:

Reviewer: use backticks to create monospace
```suggestion
. To create a node pool, enter the following information. In this example, the node pool uses agents with the `'size=large,zone notin (az1,az2)'` label:
```

Author: Addressed, ty
